util/Conventional_Monitor/image_gen/sorc/conmon_read_ps_IG.fd/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ cmake_minimum_required(VERSION 2.6)
   set_target_properties( conmon_read_ps_IG.x PROPERTIES COMPILE_FLAGS ${CONMON_READ_PS_IG_Fortran_FLAGS} )
   include_directories( ${CORE_INCS} ${NCDIAG_INCS} ) 
 
-  target_link_libraries( conmon_read_ps_IG.x ${W3NCO_4_LIBRARY} ${NCDIAG_LIBRARIES} ${NETCDF_LIBRARIES_F90} ${NETCDF_LIBRARIES} ${HDF5_Fortran_HL_LIBRARIES} ${HDF5_LIBRARIES} ${MPI_Fortran_LIBRARIES} )
+  target_link_libraries( conmon_read_ps_IG.x ${W3NCO_4_LIBRARY} ${NCDIAG_LIBRARIES} ${NETCDF_LIBRARIES_F90} ${NETCDF_LIBRARIES} ${MPI_Fortran_LIBRARIES} )
```

Comment:
Same question re. NETCDF discrepancy in the name.

Reply:
CMon cmake build does not work in master.   Does not work in this PR either.  Reach out to Ed to coordinate fix to CMon cmake build.

Reply:
Work with Ed.  Update NetCDF find_package to `find_package( NetCDF COMPONENTS C Fortran REQUIRED)`   Now cmake build from CMon level in directory tree.